code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -26,6 +26,11 @@
 	var/blood_state = BLOOD_NONE
 	var/image/blood_overlay
 
+	var/bleeding = FALSE
+	var/blood_amount = 50			// set a limit to the amount of blood it can bleed, otherwise it will keep bleeding forever and crunk the server
```

Comment:
Don't think we need this. We can just make this a function of max_health and clamp it, no?

Reply:
i don't get what you mean here